server/plugin/virtual_agent.go

```diff
@@ -85,15 +86,46 @@ type TopicPickerControl struct {
 }
 
 type OutputCard struct {
-	UIType string `json:"uiType"`
-	Group  string `json:"group"`
-	Data   string `json:"data"`
+	UIType       string `json:"uiType"`
+	Group        string `json:"group"`
+	Data         string `json:"data"`
+	TemplateName string `json:"templateName"`
 }
 
-type OutputCardData struct {
-	Subtitle string `json:"subtitle"`
-	Title    string `json:"title"`
-	URL      string `json:"url"`
+type OutputCardRecordData struct {
+	SysID            string         `json:"sys_id"`
+	Subtitle         string         `json:"subtitle"`
+	DataNowSmartLink string         `json:"dataNowSmartLink"`
+	Title            string         `json:"title"`
+	Fields           []RecordFields `json:"fields"`
```

Comment:
```suggestion
	Fields           []*RecordFields `json:"fields"`
```
See if you can use this.